CONTRIBUTING.md

```diff
@@ -12,7 +12,15 @@ Table of Contents
 
 ## Orientation
 
-We have lots more helpful information for users and developers on the [Mixxx wiki](https://mixxx.org/wiki/doku.php/start), including [build instructions](https://mixxx.org/wiki/doku.php/start#compile_mixxx_from_source_code).
+We have lots more helpful information for users and developers on the [Mixxx wiki](https://mixxx.org/wiki/doku.php/start) and elsewhere, including [build instructions](https://mixxx.org/wiki/doku.php/start#compile_mixxx_from_source_code).
+
```

Comment:
should we update this to link to github directly?

Reply:
Updated :+1: Thanks for spotting this! The latter link's fragment specifier was actually not pointing to where it should, which has also been fixed now.